### PR TITLE
catalog v1

### PR DESCRIPTION
**Pull request by Sergey8793**
https://Sergey8793.github.io/layout_catalog/
https://Sergey8793.github.io/layout_catalog/report/html_report/

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,32 @@
-body {
+@import "./blocks/header.scss";
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
+
+:root {
+  --blue: #00acdc;
```
Comment (witflash) [inline src/styles/main.scss:6]: It's better to use prefixes for variables:
```
--color-blue
--color-gray
...
```

File: src/styles/blocks/card.scss [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-name {
```
Comment (witflash) [inline src/styles/blocks/card.scss:58]: Don't divide an element name. Use full name:
```
&__price {

&__price-name {

&__price-cost {
```

**Review by witflash — CHANGES_REQUESTED** [review]
Good job!
Pay attention to the comments.

File: src/styles/blocks/card.scss (hunk 2 above)
Comment (Sergey8793) [reply src/styles/blocks/card.scss]: Tests don't pass

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    margin-right: 50px;
+  }
+  
+  &:nth-last-child(-n + 4) {
+    margin-top: 48px;
+  }
+
+  &__card {
+    display: flex;
+    flex-direction: column;
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/card.scss:12]: move this to `.card`

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: var(--color-black);
+  }
+  
+  &__code {
+    font-weight: 400;
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/card.scss:40]: it's default value, check in other places

File: src/styles/blocks/card.scss [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    border-radius: 5px;
+    border: 1px solid #ccc transparent;
+  }
+
+  &__link:hover {
+    background: #fff;
+    border: 1px solid var(--color-blue);
+    border-radius: 5px;
+    color: var(--color-blue);
+    height: 38px;
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/card.scss:94]: why do you need this?

File: src/styles/blocks/header.scss [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+.is-active {
+  color: var(--color-blue);
+
+  &::after {
+    content: "";
+    display: block;
+    border-bottom: 4px solid var(--color-blue);
+    border-radius: 8px;
+    position: relative;
+    top: 18.5px;
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/header.scss:47]:
```suggestion
    position: absolute;
    bottom: 0;
```

File: src/styles/blocks/stars.scss [hunk 7]
```diff
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-repeat: no-repeat;
+    background-image: url(/images/star.svg);
+    background-position: center;
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/stars.scss:9]: combine into `background` property

File: src/styles/main.scss [hunk 8]
```diff
@@ -1,3 +1,32 @@
...
+body,
+ul,
+p,
+h1 {
   margin: 0;
+  padding: 0;
+}
+
+.main {
+  width: 950px;
```
Comment (ivanchuk-yurii) [inline src/styles/main.scss:28]: calculate this basing on card size

File: src/index.html [hunk 9]
```diff
@@ -5,8 +5,442 @@
...
+          class="container__card card" 
+          data-qa="card"
+        >
+          <img 
+            src="images/imac.jpeg" 
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            class="card__img"
+          >
+  
+          <h1 class="card__title">
```
Comment (ivanchuk-yurii) [inline src/index.html:118]:
```suggestion
          <h2 class="card__title">
```
`h1` can be only one on page

File: src/index.html [hunk 10]
```diff
@@ -5,8 +5,442 @@
...
+          >
+            Buy
+          </a>
+        </div>
+      </div>
+
+      <div class="container">
+        <div 
+          class="container__card card" 
+        >
```
Comment (ivanchuk-yurii) [inline src/index.html:154]:
```suggestion
        <div class="container__card card">
```

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

**Comment by ivanchuk-yurii** [comment]
you should pass all tests

File: src/styles/blocks/header.scss (hunk 6 above)
Comment (Sergey8793) [reply src/styles/blocks/header.scss:47]: Does not work

**Comment by Sergey8793** [comment]
main.sccs

**Comment by Sergey8793** [comment]
main.scss

  padding: 50px 0 73px;
  margin: 0 auto;

There should only be padding, but padding: 50px auto 73px; don`t work

File: src/styles/blocks/card.scss (hunk 2 above)
Comment (witflash) [reply src/styles/blocks/card.scss]: Ok, so let it as it

File: src/styles/blocks/header.scss (hunk 6 above)
Comment (Viktor-Kost) [reply src/styles/blocks/header.scss:47]: Here is a deeper stuff. 
You need to add to your link `&__nav-link` a rule `position: relative;`

Then your pseudo-element` &::after` will be a little bit different:
Add `position: absolute; bottom: 0; background-color: var(--color-blue); width: 100%; height: 4px; `
Delete `top: 18.5px; border-bottom: 4px solid var(--color-blue);`

File: src/styles/blocks/card.scss (hunk 5 above)
Comment (Viktor-Kost) [reply src/styles/blocks/card.scss]: is not fixed. 
Use padding instead of hardcode height

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Hi!)
Pay attention to previous comments. I left them unresolved.

Also, if you need any help - ask at fe_chat

**Comment by Sergey8793** [comment]
> Hi!) Pay attention to previous comments. I left them unresolved.
> 
> Also, if you need any help - ask at fe_chat

**Comment by Sergey8793** [comment]
I asked but they told me everything was fine

**Pull request closed without merging**